### ISSUE TITLE
PublicKey is not Buffer

### ISSUE THREAD

**Issue by ghost** [doublejumptokyo/fcl-kms-authorizer#13]
When trying to get the public flow key to register on the faucet, we get the error `PublicKey is not Buffer`

```
  const authorizer = new KmsAuthorizer(
    {
      region: 'region',
      credentials: {
        accessKeyId: 'key',
        secretAccessKey: 'secret key',
      },

    },
    'XXXX-XXXX-XXXX-XXX',
  );
  
  console.log(await authorizer.getFlowPublicKey())

```

The type actually being returned is `Uint8Array(88)`

**Comment by ghost** [comment]
Downgrading to 0.0.9 fixes the above issue but the key length that is returned is 146 chars long instead of 128. Any advice or AWS config that needs to be applied?

Key Spec: ECC_SECG_P256K1
Key Usage: Sign and verify

**Comment by avcdsld** [comment]
@bencresswell  I apologize for the delay in replying. 🙏 
This is a bug and I will fix it.

There was a difference in the AWS Client version between versions 0.0.9 and 0.0.10, and testing here was lacking.

**Comment by avcdsld** [comment]
@bencresswell This issue has been resolved, please try v0.0.11.

> Downgrading to 0.0.9 fixes the above issue but the key length that is returned is 146 chars long instead of 128. Any advice or AWS config that needs to be applied?

Please use `getPublicKey()` instead of `getFlowPublicKey()`, as described in the README.
The result of `getFlowPublicKey()` is a form of key that can be used when creating an account using a Flow transaction. You don't need to use it on Testnet Faucet page.
